src/time_manager.cpp

```diff
@@ -12,7 +12,7 @@
 namespace Time {
 TimeManager::TimeManager(hwlib::pin_oc &scl, hwlib::pin_oc &sda)
     : scl(scl), sda(sda), realTimeClock(hwlib::i2c_bus_bit_banged_scl_sda(scl, sda)) {
-    setTime(0);
+    TimeManager::setTime(RTCTime(0));
```

Comment:
Why is TimeManager explicitly noted in this subroutine invocation? `setTime(RTCTime(0))` should be enough, unless setTime is ambiguous with functions in different class scopes that are inherited.